paasta_tools/kubernetes_tools.py

```diff
@@ -522,7 +522,7 @@ def __init__(
         self.deployments = kube_client.AppsV1Api(self.api_client)
         self.core = kube_client.CoreV1Api(self.api_client)
         self.policy = kube_client.PolicyV1beta1Api(self.api_client)
-        self.apiextensions = kube_client.ApiextensionsV1beta1Api(self.api_client)
+        self.apiextensions = kube_client.ApiextensionsV1Api(self.api_client)
```

Comment:
Since we also use `update_crds` for setup_kubernetes_crd, and these come from yelpsoa-configs, there will be a short period where things break until the CRD definitions in yelpsoa are updated to also use V1 since they currently match what kube_client.apiextensions is using.

I think it would be a bit nicer if we could make this more flexible so paasta reads in what version CRD owners are using in their files.

But if we just want to support one version as you're doing here, we likely will need to pin paasta & roll out slowly, plus work with CRD owners to let them know as we update their configs in yelpsoa.